clevertap-core/src/test/java/com/clevertap/android/sdk/inapp/CTInAppBaseFragmentTest.kt

```diff
@@ -143,9 +143,13 @@ class CTInAppBaseFragmentTest {
         verify {
             inAppListener.inAppNotificationActionTriggered(
                 inAppNotification = any(),
-                action = any(),
+                action = match { action ->
```

Comment:
This is not relevant to the HTML template change. I can remove it from the PR, but it would be good to have this small improvement merged.